jenkins-build-driver/src/main/java/org/jboss/pnc/jenkinsbuilddriver/JenkinsBuildDriver.java

```diff
@@ -42,11 +44,22 @@ public String getDriverId() {
     public boolean canBuild(BuildType buildType) {
         return BuildType.JAVA.equals(buildType);
     }
+    
+    public void init(RunningEnvironment runningEnvironment) {
+        try {
+            this.isCrumbUsed = jenkinsServerFactory.isJenkinsServerSecuredWithCSRF(runningEnvironment.getJenkinsUrl());    
+        } catch ( BuildDriverException bde) {
```

Comment:
Yes, if something is wrong with Jenkins server it would be reported later, that is just checking for flag, there is not really reason to assume that this is significant for following process.